code/game/objects/items/RCD.dm

```diff
@@ -1237,6 +1231,8 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 	. = ..()
 	if(.)
 		return
+	if(!usr.canUseTopic(src, be_close = TRUE))
```

Comment:
You should now be able to safely remove this

Reply:
Um did you read the comment in your last review?

> Nope it didnt work :(. You definitly caught a bug thanks for that but ghosts can still have their fun with the RCD and the console still needs ui_host() to open

RCD & Plumbing RCD still requires canUseTopic() else ghosts can still mess with it. I resolved that conversation because i made the change you asked me too but not because it solved this problem

Reply:
Going to take a look at this again tomorrow then because this is 100% not necessary, nothing else needs this